### PR TITLE
Issue/30 add option to use module in place

### PR DESCRIPTION
**Pull request by kazaamjt**

### CODE REVIEW

File: tests/test_options.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    shutil.move("model", "testmodule/model")
+    shutil.move("module.yml", "testmodule/module.yml")
+    shutil.move("plugins", "testmodule/plugins")
+    shutil.move("tests", "testmodule/tests")
+    os.chdir("testmodule")
+    pytest_inmanta.plugin.CURDIR = (os.getcwd())
+
+    result = testdir.runpytest("tests/test_resource_run.py", "--use-module-in-place")
+
+    result.assert_outcomes(passed=1)
```
Comment (wouterdb) [inline tests/test_options.py]: this doesn't seem to test if it is in place or not

File: pytest_inmanta/plugin.py [hunk 2]
```diff
@@ -138,13 +146,14 @@ def project_shared(request):
         fd.write("""name: testcase
 description: Project for testcase
 repo: ['%(repo)s']
-modulepath: libs
+modulepath: ['%(modulepath)s']
 downloadpath: libs
-""" % {"repo": "', '".join(repos)})
+""" % {"repo": "', '".join(repos), "modulepath": "', '".join(modulepath)})
```
[the remark this answers is not in the document]
Comment (kazaamjt) [reply pytest_inmanta/plugin.py]: What do you propose? I simply went with what was already there for the repos.

File: tests/test_options.py (hunk 1 above)
Comment (kazaamjt) [reply tests/test_options.py]: resolved

File: pytest_inmanta/plugin.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (wouterdb) [reply pytest_inmanta/plugin.py]: ok, in line with wath is there, then it is ok

**Pull request merged**